### DISCUSSION TITLE
Deformation results are abnormal

### DISCUSSION THREAD

**Discussion by amosaha** [idaholab/moose#20724] [Q&A Modules: Solid mechanics] [answered]
I made a thermal expansion model of a fuel, but one of the fuels cannot be deformed. What is the problem?
![image](https://user-images.githubusercontent.com/93416273/161876137-cd353fc4-8dba-4777-b12c-63242a55bbb5.png)

and this is my input file.
[cchf2.txt](https://github.com/idaholab/moose/files/8422832/cchf2.txt)

**Comment by GiudGiud** [comment, marked as answer]
hello

We're going to need to see the mesh as well, I dont see anything wrong in the input file, but maybe the blocks or the surface definitions are wrong.

Guillaume

**Reply by amosaha** [reply]
I have checked  the mesh many times. Thank you for your help！
[cchf.zip](https://github.com/idaholab/moose/files/8423339/cchf.zip)

**Reply by GiudGiud** [reply]
i think your material defintions are wrong?
For example the claddings are 1 4 7 but really it should be 1 3 5 ?

I would just not use the block IDs. Use the block names that way you will be sure to get it right

**Comment by amosaha** [comment]
Unfortunately, after I fixed the problem, it still did not solve the problem.






发自我的iPhone


------------------ Original ------------------
From: Guillaume Giudicelli ***@***.***&gt;
Date: Wed,Apr 6,2022 11:45 AM
To: idaholab/moose ***@***.***&gt;
Cc: amosaha ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [idaholab/moose] Deformation results are abnormal (Discussion #20724)





 
i think your material defintions are wrong?
 For example the claddings are 1 4 7 but really it should be 1 3 5 ?
 
I would just not use the block IDs. Use the block names that way you will be sure to get it right

**Reply by GiudGiud** [reply]
Can you have the updated input file here?

**Reply by amosaha** [reply]
Input file is here. I think there may be some problems when moose calculates the deformation of multiple objects
[cchfupdata.txt](https://github.com/idaholab/moose/files/8438612/cchfupdata.txt)
.

**Reply by GiudGiud** [reply]
you have `    block = ''2 4 6' ` instead of `    block = '2 4 6'`
in your conduction material

can you please check if that is the problem?
